src/pages/Registration/MemberForm.jsx

```diff
@@ -90,11 +90,11 @@ const MemberForm = ({ control, index }) => {
         </Grid>
         <Grid item xs={4}>
           <FormControl fullWidth>
-            <InputLabel id="gender">Gender *</InputLabel>
+            <InputLabel id="gender">{t("gender")} *</InputLabel>
             <Select {...genderField} required labelId="gender" label="Gender">
-              <MenuItem value="Male">Male</MenuItem>
-              <MenuItem value="Female">Female</MenuItem>
-              <MenuItem value="Other">Others</MenuItem>
+              <MenuItem value="Male">{t("male")}</MenuItem>
+              <MenuItem value="Female">{t("female")}</MenuItem>
+              <MenuItem value="Other">{t("others")}</MenuItem>
             </Select>
           </FormControl>
           <p error={genderFieldState.invalid} style={{ color: "red" }}>
```

Comment:
Fix the custom error messages also

Reply:
These errors are of react-hook-form. Cannot translate them

Can only translate the ones in zod validation

Reply:
What kind of errors does this throw mainly?

Reply:
Caused by incorrect filling of form according to rules of hook-form. Like NaN value, empty value or can be something else

Reply:
Got it so these are the errors which one must understand 
If the errors can't be shown in Hindi then there's no point of translation
See if something can be done about it